css/properties/background-blend-mode.json

```diff
@@ -12,7 +12,8 @@
               "version_added": null
             },
             "edge": {
-              "version_added": "18"
+              "version_added": null,
```

Comment:
Probably `false` because `null` means unknown.

Reply:
Thanks @saschanaz , fixed it. However I accidentally added a merge commit to this PR. I'll close and reopen with a clean history.